server/store.go

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/pkg/errors"
+
+	"github.com/mattermost/mattermost-plugin-zoom/server/zoom"
+)
+
+const (
+	zoomStateKeyPrefix = "zoomuserstate"
```

Comment:
By using a fixed prefix here instead, I think we can avoid having a bunch of excess state entries in the KV store due to incomplete connection attempts from users.